Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -1831,19 +2072,333 @@ def export_pcap_file_command(client: Client, args: Dict) -> List:
+    """
+    Args:
+        client(Client): client - A McAfeeNSM client.
+        args(Dict): - The function arguments.
+    Returns: A CommandResult object with a success or failure message.
+    """
+    device_id = arg_to_number(args.get('device_id'))
+    domain_id = arg_to_number(args.get('domain_id'))
+    pre_firewall_policy = args.get('pre_firewall_policy_name')
+    post_firewall_policy = args.get('post_firewall_policy_name')
+
+    response = client.assign_device_policy_request(domain_id=domain_id, device_id=device_id,
+                                                   pre_firewall_policy=pre_firewall_policy,
+                                                   post_firewall_policy=post_firewall_policy
+                                                   )
+
+    return CommandResults(
+        readable_output='Policy assigned successfully.',
+        raw_response=response
+    )
+
+
+def list_device_policy_command(client: Client, args: Dict) -> CommandResults:
+    """
+
+    Args:
+        client(Client): client - A McAfeeNSM client.
+        args(Dict): - The function arguments.
+
+    Returns:
+        CommandResults: A CommandResult object with a list of device policies.
+    """
+    device_id = arg_to_number(args.get('device_id'))
+    domain_id = arg_to_number(args.get('domain_id'))
+    limit = arg_to_number(args.get('limit', 50))
+    all_results = argToBoolean(args.get('all_results', False))
+    response = client.list_device_policy_request(domain_id=domain_id, device_id=device_id)
+
+    all_policies = (response.get('policyAssignmentsList') if all_results
+                    else
+                    response.get('policyAssignmentsList', {})[:limit])
+    # Capitalize the keys of the policies list and keeping the rest of the keys as is.
+    capitalize_policies = []
+    for policy in all_policies:
+        policy = {k[:1].upper() + k[1:]: v for k, v in policy.items()}
+        capitalize_policies.append(policy)
+
+    readable_output = tableToMarkdown(
+        name='Device policy List', t=capitalize_policies, removeNull=True
+    )
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='NSM.DevicePolicy',
+        outputs=capitalize_policies,
+        raw_response=response
+    )
+
+
+def assign_interface_policy_command(client: Client, args: Dict) -> CommandResults:
+    """
+    Args:
+        client(Client): - A McAfeeNSM client.
+        args(Dict): - The function arguments.
+    Returns: A CommandResult object with a success or failure message.
+    """
+    domain_id = arg_to_number(args.get('domain_id'))
+    interface_id = arg_to_number(args.get('interface_id'))
+    firewall_policy = args.get('firewall_policy_name')
+    firewall_port_policy = args.get('firewall_port_policy_name')
+    ips_policy = args.get('ips_policy_name')
+    custom_policy_json = json.loads(args.get('custom_policy_json'))  # type: ignore
```

Comment:
Maybe a default return value instead of type ignore? 
```suggestion
    custom_policy_json = json.loads(args.get('custom_policy_json', {}))
```